relay/kusama/src/lib.rs

```diff
@@ -935,6 +936,13 @@ impl pallet_identity::Config for Runtime {
 	type Slashed = Treasury;
 	type ForceOrigin = EitherOf<EnsureRoot<Self::AccountId>, GeneralAdmin>;
 	type RegistrarOrigin = EitherOf<EnsureRoot<Self::AccountId>, GeneralAdmin>;
+	// TODO:(PR#159) check adds bellow and remove this comment!
+	type OffchainSignature = Signature;
+	type SigningPublicKey = <Signature as Verify>::Signer;
+	type UsernameAuthorityOrigin = EnsureRoot<Self::AccountId>;
+	type PendingUsernameExpiration = ConstU32<{ 7 * DAYS }>;
+	type MaxSuffixLength = ConstU32<7>;
+	type MaxUsernameLength = ConstU32<32>;
```

Comment:
@joepetrowski Joe, as an author of https://github.com/paritytech/polkadot-sdk/pull/2651, can you please check this identity stuff for Kusama and for Polkadot?

Reply:
Looks good for both 👍